doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1037 @@
+
+Wi-Fi
+-----
+
+* Added support for nRF700x DK that supports Wi-Fi SoC, which implements IEEE 802.11 protocols.
+  For more information, see :ref:`ug_nrf70`.
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in your application.
+
+RF Front-End Modules
+====================
+
+* Fixed a build error that occurred when building an application for nRF53 SoCs with Simple GPIO Front-End Module support enabled.
+* Added the :kconfig:option:`CONFIG_MPSL_FEM_ONLY` Kconfig option that allows the :ref:`nrfxlib:mpsl_fem` API to be used without other MPSL features.
+  The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
+  You cannot use other MPSL features when this option is enabled.
+* Added the possibility to add custom models which split the requested TX power between the power on SoC output and the FEM gain in a way desired by the user.
+
+Secure Partition Manager (SPM)
+==============================
+
+* Deprecated Secure Partition Manager (SPM) and its the Kconfig option :kconfig:option:`CONFIG_SPM`.
+  It is replaced by the Trusted Firmware-M (TF-M) as the supported trusted execution solution.
+  See :ref:`ug_tfm` for more information about the TF-M.
+* Updated files related to TF-M and SPM, included in samples and applications with non-secure build targets, to no longer include SPM as an alternative to TF-M.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+* All applications running on non-secure boards are documented to use TF-M as the trusted execution solution by default.
+  SPM is now deprecated.
+* In applications running on Nordic Thingy:91, changed the PWM frequency of the pwmleds device from 50 Hz to 125 Hz.
+
+nRF9160: Asset Tracker v2
+-------------------------
+
+* Added:
+
+  * :ref:`motion_impact_detection` using the ADXL372 accelerometer.
+  * The following Kconfig options to set the threshold and timeout values:
+
+      * :ref:`CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD>`
+      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD>`
+      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS <CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS>`
+
+  * Support for full modem FOTA updates for nRF Cloud builds.
+
+* Updated:
+
+  * The application to use :ref:`TF-M <ug_tfm>` enabled by default.
+  * Data sampling method.
+    It is now performed when the device detects both activity and inactivity in passive mode, notified by the :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` event of the :ref:`sensor module <asset_tracker_v2_sensor_module>`.
+  * ``CONFIG_MODEM_NEIGHBOR_SEARCH_TYPE`` Kconfig option.
+  * The default value of the GNSS timeout in the application's :ref:`Real-time configurations <real_time_configs>`.
+    It is now set to 30 seconds.
```

Comment:
I believe this overlaps with Line 253. Adding the 30 seconds as the default value there could allow removing these lines.